### ISSUE TITLE
Call to Action: Copy Writing

### ISSUE THREAD

**Issue by jalcine** [blkops/blkops.github.io#3]
So we'd need to get some copy writing for the site. This issue's here to serve as a permanent forum of sorts (outside of Slack since it's viewable without authentication). What are some things we need (feel free to edit + update)?
- [x] < 100 word description of what `blkops` aims to achieve, in a mission statement / objective like prose.
- [x] A list of FAQ we can use when onboarding other people into the fold.
- [ ] Description for the Rise [mobile](https://github.com/blkops/rise-ionic) and [API](https://github.com/blkops/rise-api) layers.
- [ ] With consent, the list of organizations we're working with to provide services for.

@blkops/owners: What do y'all think?

**Update 2015-02-07 02:23 PM EST**: @southgraphic has a _lot of content_ that can be used for the description.

**Comment by ohpeacock** [comment]
Perhaps a projects page as well, listing all of the projects that we've done, once we've completed one (We probably don't want to include information about projects that we're currently working on... or do we?). I also think that having a contact us page is important as well, but that can be rectified by providing that information in the FAQs (would we want a generic email for that? Or if people are interested in collaborating we could have people place an issue on the blkops GitHub repo that they're interested in working on... But then they'd be required to have a GitHub account...)

I know that @southgraphic (aka. mouthsmart) created a Mission Statement [here](https://github.com/blkops/blkops.github.io/wiki/Mission-Statement) in our wiki. I think it can serve as a great start, but it would need to be sized down substantially if we want less than 100 words. 

As far as FAQ content, I'm thinking: How do I get involved? Who can get involved? How can I see the progress of Blkops' current project/What is Blkops working on now? I have some feedback for a tool, who do I contact?

**Comment by jalcine** [comment]
> Perhaps a projects page as well, listing all of the projects that we've done, once we've completed one
> (We probably don't want to include information about projects that we're currently working on... or do > we?)

Haha, probably only the publically facing projects we have out here! So far, that's Rise and the list of curated reading material.

> I also think that having a contact us page is important as well, but that can be rectified by providing 
> that information in the FAQs 

Fair point. I'd add to that by pointing out that the question on contact could then  _point to_ the contact page, which could handle the actual work of sending an email and block out spam.

> Or if people are interested in collaborating we could have people place an issue on the blkops
> GitHub repo that they're interested in working on... But then they'd be required to have a GitHub
> account...)

Yeah, I guess for collaborators, we'd recommend them to open an issue; anyone else to send an email (or we can have yet another contact form that collects feature requests nicely).

> I know that @southgraphic (aka. mouthsmart) created a Mission Statement here in our wiki. It think
> it can serve as a great start, but it would need to be sized down substantially if we want less than 
> 100 words. 

I chose 100 because it can fit relatively well on most viewports without too much tampering of kerning, sizes and what not. However, he did go _really_ deep and I think we can actually have it take the two sentences and have it link to a page that'd go into more detail. Thanks @southgraphic!

> As far as FAQ content, I'm thinking:

All of these are dope, probably could hammer this out into a list now :)

Right now, I'm fixing up some stuff with [my federated services](http://jalcine.me/weblog/going-sovereign/) but I'll be back in a few hours to hack out on this a little.

**Comment by ohpeacock** [comment]
> Haha, probably only the publically facing projects we have out here! So far, that's Rise and the list of curated reading material.

Agreed haha. At the moment, I'm thinking a projects page with about a paragraph or so (a smallish snippet) of what the project's about. Once projects are released, they could probably get their own pages. 

> Fair point. I'd add to that by pointing out that the question on contact could then point to the contact page, which could handle the actual work of sending an email and block out spam.

Good point! We can definitely have a link to a contact page in the FAQs. 

> Yeah, I guess for collaborators, we'd recommend them to open an issue; anyone else to send an email (or we can have yet another contact form that collects feature requests nicely).

Oooooo, I like the sound of organized feature requests! 

> I chose 100 because it can fit relatively well on most viewports without too much tampering of kerning, sizes and what not. However, he did go really deep and I think we can actually have it take the two sentences and have it link to a page that'd go into more detail. Thanks @southgraphic!

That could work! Good job thinking ahead about word size manipulation (that's never fun...).

I can get started on the FAQs page now (I'll be going back and forth between working on this and some hw). For now, I'll place it in the master branch in a folder called pages (and please don't hesitate to move things around if you think my organization system is weird! haha). I guess we can create these as markdown pages now and then migrate them over to be integrated with the jekyll structure. 

**Edit:** Here's the direct link to the FAQs: https://github.com/blkops/blkops.github.io/blob/master/pages/FAQs.md
